_data-prepper/simple-pipelines.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Simple pipelines
+nav_order: 12
+---
+
+# Simple pipelines
+
+This tutorial walks you through creating a simple Data Prepper pipeline and how to configure it.
+
+## Installation
+
+First, install Data Prepper with Docker using the following command:
+
+```
+docker pull opensearchproject/data-prepper:latest
+```
+
+## Simple configuration
+
+To help you get started, we have a simple configuration which you can use to run Data Prepper. It generates random strings and writes them to `stdout`. 
+
+Create a Data Prepper configuration file, `data-prepper-config.yaml` and include the following line:
+
+```
+ssl: true
+```
+
+Create a Data Prepper pipeline file, `pipelines.yaml`, with:
+
+```
+simple-sample-pipeline:
+  workers: 2
+  delay: "5000"
+  source:
+    random:
+  sink:
+    - stdout:
+```
+
+## Run Data Prepper
+
+Below are examples for running Data Prepper from the Docker image. If you built from source, you need to make some modifications to the example commands.
```

Comment:
```suggestion
Following are examples of running Data Prepper from the Docker image. If you built from source, you need to make some modifications to the example commands.
```

Reply:
Let's clarify what we mean by "from source". From what source? Use "below" only for physical space or screen descriptions, such as “the outlet below the vent” or “the button below the bar pane.” For orientation within a document, use "following" or "later."